packages/language-server/package.json

```diff
@@ -35,6 +35,7 @@
   },
   "dependencies": {
     "@neo4j-cypher/language-support": "2.0.0-next.2",
+    "@neo4j-cypher/schema-poller": "2.0.0-next.2",
```

Comment:
Why is this needed? I don't think this would work with the publishing since this package is not in npm

Reply:
my mistake